### PR TITLE
fix: robust timeout and patch split param

### PR DESCRIPTION
**Pull request by flashlab**

### CODE REVIEW

**Review by mihailefter — COMMENTED** [review]
Why do you need the timeout default value changed? It can be configured via the CLI or from the function call.

File: mutalyzer_retriever/cli.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def _parse_args(args):
         "--ref_id_start", help="reference id should start with"
     )
 
+    parser_ncbi_assemblies.add_argument(
```
[the remark this answers is not in the document]
Comment (flashlab) [reply mutalyzer_retriever/cli.py:109]: Actually I can't use `--split` in `ncbi_assemblies` subcommand like `mutalyzer_retriever ncbi_assemblies  --input downloads/ --ref_id_start NC_000011 --downloaded --split`

**Comment by flashlab** [comment]
> Why do you need the timeout default value changed? It can be configured via the CLI or from the function call.

Sorry, I missed `timeout` argument since it's not in the document. I'll close the pull request.
But the question still remains, why the program failed to extract exon and transcript feature from some kind of reference like NC_000011.9(GRCH37). Seems same issue in #12.

File: mutalyzer_retriever/cli.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mihailefter) [reply mutalyzer_retriever/cli.py:109]: You can make use of the following: `mutalyzer_retriever --split ncbi_assemblies  --input downloads/ --ref_id_start NC_000011 --downloaded`.

**Pull request closed without merging**